### PR TITLE
Add search.revvy.de, fix things that visually disturb me

### PR DESCRIPTION
**Pull request by codedipper**

### CODE REVIEW

**Comment by codedipper** [comment]
@Ahwxorg is there a problem with this pr? I saw you merged my other pr pretty quickly. This instance is fully functional now.

**Comment by Ahwxorg** [comment]
Look at the code reviews, already pinged you there :)

**Comment by codedipper** [comment]
> Look at the code reviews, already pinged you there :)

I'm sorry, I'm confused. What code reviews?

**Comment by Ahwxorg** [comment]
It should be visible now.

File: README.md
[the remark this answers is not in the document]
Comment (codedipper) [reply README.md:1]: Instances that have Tor and/or I2P are meant to go first. It doesn't really matter, you can arrange it how you want.

[the remark this answers is not in the document]
Comment (Ahwxorg) [reply README.md:1]: Alright! It's fine, I just wanted to ask why

**Review by Ahwxorg — APPROVED** [review]

**Pull request merged**